seqr/views/apis/project_api.py

```diff
@@ -254,17 +258,19 @@ def project_families(request, project_guid):
 def project_overview(request, project_guid):
     project = get_project_and_check_permissions(project_guid, request.user)
 
-    sample_models = Sample.objects.filter(individual__family__project=project)
+    sample_load_counts, sample_models = _sample_load_counts(
+        Sample, project, 'sample_type', 'dataset_type', loadedDate=TruncDate('loaded_date'),
+    )
+    rna_sample_load_counts, _ = _sample_load_counts(
+        RnaSample, project, sample_type=Value('RNA'), dataset_type=F('data_type'), loadedDate=TruncDate('created_date'),
+    )
 
     first_loaded_samples = sample_models.order_by('individual__family', 'loaded_date').distinct('individual__family').values_list('id', flat=True)
     samples = sample_models.filter(Q(is_active=True) | Q(id__in=first_loaded_samples))
     samples_by_guid = {s['sampleGuid']: s for s in get_json_for_samples(samples, project_guid=project_guid)}
 
-    sample_load_counts = sample_models.values(
-        'sample_type', 'dataset_type', loadedDate=TruncDate('loaded_date'),
-    ).order_by('loadedDate').annotate(familyCounts=ArrayAgg('individual__family__guid'))
     grouped_sample_counts = defaultdict(list)
-    for s in sample_load_counts:
+    for s in sample_load_counts + rna_sample_load_counts:
         s['familyCounts'] = {f: s['familyCounts'].count(f) for f in s['familyCounts']}
         grouped_sample_counts[f'{s.pop("sample_type")}__{s.pop("dataset_type")}'].append(s)
 
```

Comment:
Should the project_overview endpoint also return rnaSamples? I imagine we want to display this in the `RNA-seq Datasets` section of the project overview UI but I haven't gotten to the UI changes in this PR yet

Reply:
ok yeah I could be wrong but I think `sample_models` will not include any RNA samples any more so you'll have to explicitly pass them either inside `samplesByGuid` or elsewhere in the response

Reply:
so we do not want to return all the RNA models, thats an intentional choice: https://github.com/broadinstitute/seqr/pull/4242#discussion_r1681656818